tests/fixture/testnet/local/network.go

```diff
@@ -632,3 +644,48 @@ func (ln *LocalNetwork) ReadAll() error {
 	}
 	return ln.ReadNodes()
 }
+
+func (ln *LocalNetwork) AddLocalNode(w io.Writer, node *LocalNode) (*LocalNode, error) {
+	// Assume network configuration has been written to disk and is current in memory
+
+	if node == nil {
+		node = NewLocalNode("")
```

Comment:
can we add a comment on what is happening here. is this a node with no name or default or?